extensions/functions_datetime.yaml

```diff
@@ -193,3 +203,66 @@ scalar_functions:
           - value: interval_year
           - value: interval_year
         return: boolean
+  - name: "assume_timezone"
+    description: Convert local time to UTC using given local time's timezone.
+    impls:
+      - args:
+          - options: [ TIMEZONE ]
+            name: Timezone of provided local time.
+            required: true
+          - value: timestamp
+        return: timestamp_tz
+  - name: "strptime"
+    description: Parse string into timestamp/date/time using provided format.
+    impls:
+      - args:
+          - options: [ FORMAT ]
+            name: Expected format of the input timestamp/date/time.
+            required: true
+          - value: string
+        return: timestamp
+      - args:
+          - options: [ FORMAT ]
```

Comment:
This isn't allowed: two variations of the same function with the same input argument types. Both above and below are option + string.

Reply:
Removing.